styles.css

```diff
@@ -0,0 +1,365 @@
+body {
+  font-family: Public Sans;
+  font-size: 18px;
+}
+
+header {
+  width: 100%;
+  height: 70px;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  font-weight: 600;
+  color: #919095;
+}
+
+.header_navbar li {
+  display: inline;
+  padding: 20px;
+}
+
+.header_navbar li:hover {
+  transition: 100ms;
+  border-bottom: 5px solid transparent;
+  border-image: linear-gradient(to right, #31cf72, #2bb9d1);
+  border-image-slice: 1;
+}
+
+.request_button {
+  height: 45px;
+  width: 170px;
+  background-image: linear-gradient(to right, #31cf72, #2bb9d1);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 25px;
+  color: #ffffff;
+  font-weight: 700;
+}
+
+.request_button:hover {
+  opacity: 0.5;
+  transition: 100ms;
+}
+
+.hidden_desktop {
+  display: none;
+}
+
+h1 {
+  font-size: 45px;
+}
+
+.banner {
+  background-color: #fafafa;
+}
+
+.banner_image_desktop {
+  height: 600px;
+  width: 150%;
+  object-fit: cover;
+  object-position: 50% 113%;
+}
+
+#benefits {
```

Comment:
Tutaj lepiej dodać klasę .benefits niż odwoływać się po #.
Ogólnie id w html powinno służyć do wybierania elementów przez js, i jeśli możemy to lepiej stylować po klasach